### PR TITLE
[MRG+1] MEP12 text alignment example

### PR DESCRIPTION
**Pull request by dstansby**
Another documentation update!

### CODE REVIEW

File: examples/text_labels_and_annotations/text_alignment.py [hunk 1]
```diff
@@ -47,24 +49,24 @@
         verticalalignment='top',
         transform=ax.transAxes)
 
-ax.text(left, 0.5*(bottom + top), 'right center',
+ax.text(left, 0.5 * (bottom + top), 'right center',
```
Comment (thisch) [inline examples/text_labels_and_annotations/text_alignment.py]: I'm -1 on this and the following changes. Note that PEP8 AFAIK does not recommend to add whitespace before and after this operator.

Comment (anntzer) [inline examples/text_labels_and_annotations/text_alignment.py]: Actually it does, see https://www.python.org/dev/peps/pep-0008/#other-recommendations.

File: examples/text_labels_and_annotations/text_alignment.py [hunk 2]
```diff
@@ -1,22 +1,24 @@
 """
+===================
+Precise text layout
+===================
+
 You can precisely layout text in data or axes (0,1) coordinates.  This
 example shows you some of the alignment and rotation specifications to
-layout text
+layout text.
```
Comment (anntzer) [inline examples/text_labels_and_annotations/text_alignment.py]: "for layout text"?  (nonnative speaker here)

File: examples/text_labels_and_annotations/text_alignment.py (hunk 1 above)
Comment (thisch) [inline examples/text_labels_and_annotations/text_alignment.py]: It says 'Use your own judgment;'

Comment (anntzer) [inline examples/text_labels_and_annotations/text_alignment.py]: "If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies)."

File: examples/text_labels_and_annotations/text_alignment.py (hunk 2 above)
Comment (QuLogic) [inline examples/text_labels_and_annotations/text_alignment.py]: "for text layout"

**Comment by anntzer** [comment]
Thanks!

File: examples/text_labels_and_annotations/text_alignment.py (hunk 1 above)
Comment (thisch) [inline examples/text_labels_and_annotations/text_alignment.py]: In the above line the operator with the lowest priority is the plus operator around which whitespace is added. PEP8 does not say anything about adding whitespace to all operators independent of their priorities.

**Pull request merged**